the-collab-lab/tcl-44-smart-shopping-list
# Font reset and loading image

## Description

This PR adjusts fonts for consistency, adds a tittle and adds a loading icon.

- I applied font color to the whole app with the class `text-stone-800`.
- I added font size `text-xl` to paragraphs in` Home.js` to have consistency across the app.
- I added the tittle **My Items** to `List.js`.
- In `List.js`, I replaced the strig `Loading...` with a loading icon. I used the class `animate-spin` to make the icon spin.

## Related Issue

Closes #57 

## Acceptance Criteria

- When the app is loading a loading icon should appear instead of the string "Loading..."
- The whole app should be consistent in terms of font-color and font-size for the tittles and paragraphs.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|   ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |




## Testing Steps / QA Criteria

- From your terminal, pull down this branch with `git pull origin  jm-font-reset-and-loading-image` and check that branch out with `git checkout  jm-font-reset-and-loading-image`
- run `npm start`
